lib/ramble/ramble/workspace/workspace.py

```diff
@@ -292,6 +292,18 @@ def licenses_file(path):
     return get_yaml_filepath(path, licenses_file_name)
 
 
+def all_config_files(path):
+    """Returns path to all yaml files in workspace config directory"""
+    config_files = []
+
+    config_path = os.path.join(path, workspace_config_path)
+    for f in os.listdir(config_path):
+        if f.endswith(".yaml"):
```

Comment:
Is this right? Do we only care about `.yaml` or do we also need to compare `.tpl` files?

Reply:
I was thinking `--all` was all config files, but that's a good point. It's probably better to have it be all of the above, including template files

Reply:
Yea to me it feels like something might be "wrong" if `ramble workspace edit` gives you more files than `--all`

Reply:
OK, I've updated it. This method still only returns the yaml files because there's already an 'all template' method, so I combined the two in cmd instead of here.